karma-dev.conf.js

```diff
@@ -1,27 +0,0 @@
-module.exports = config =>
-  config.set({
-    basePath: "",
-    frameworks: ["jspm", "jasmine"],
-    files: [".tmp/dev-bundle.js", "jspm_packages/system-polyfills.js"],
-    jspm: {
-      config: "jspm.config.js",
-      browser: "jspm.test.js",
-      loadFiles: [
-        "src/test-setup.ts",
-        {pattern: "src/**/*.spec.ts", watched: false, served: false}
-      ]
-    },
-    preprocessors: {
-      ".tmp/dev-bundle.js": ["sourcemap"]
-    },
-    reporters: ["mocha"],
-    mochaReporter: {
-      output: 'minimal'
-    },
-    port: 9876,
-    colors: true,
-    logLevel: config.LOG_WARN,
-    autoWatch: true,
-    browsers: ["PhantomJS"],
-    singleRun: false
-  });
```

Comment:
Although most of the default values should be ok for now, I'm still not comfortable with not being able to customize or extend the karma configuration used for running the tests.  We both know that especially dealing with karma-jspm quickly runs into requiring modication of this configuration. It should with the template in its current state, but even slight modification or extension by anyone using it might fire up this issue.
Imho, not providing something to add additional configuration for karma is a **blocker**.